src/pages/Jongmin/Main/Components/comment.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import CommentList from './commentList';
+import './comment.scss';
+
+class Comment extends React.Component {
+  componentDidMount = () => {
+    this.props.componentDidMount();
+  };
+
+  handleCommentValue = () => {
+    this.props.handleCommentValue(this.props.commentValue);
+  };
+
+  addComment = e => {
+    e.preventDefault();
+    const { commentList, commentValue } = this.state;
+    this.setState({
+      commentList: [
+        ...commentList,
+        {
+          id: commentList.length + 1,
+          userName: 'jongmin_8910',
+          content: commentValue,
+          isLiked: false,
+        },
+      ],
+      commentValue: '',
+    });
+  };
+
+  deleteReply = index => {
+    console.log('클릭');
+    const { commentList } = this.state;
+    const otherReplys = commentList.filter(element => {
+      return element.id !== index;
+    });
+    this.setState({ commentList: otherReplys });
+  };
+
+  changeLike = id => {
+    const commentList = [...this.state.commentList];
+
+    commentList.filter(ele => {
+      if (ele.id === id) {
+        commentList[id - 1].isLiked = !commentList[id - 1].isLiked;
+      }
+    });
+    this.setState({ commentList: commentList });
+  };
+
+  render() {
+    const { commentList, commentValue } = this.state;
+    return (
+      <div className="commentArea">
+        <ul>
+          {commentList.map(comment => {
+            return (
+              <CommentList
+                key={comment.id}
+                commentId={comment.id}
+                UserId={comment.userName}
+                comment={comment.content}
+                isLiked={comment.isLiked}
+                deleteComment={this.deleteReply}
+                changelike={this.changeLike}
+                blackHeart={this.state.blackHeart}
+                redHeart={this.state.redHeart}
+              />
+            );
+          })}
+        </ul>
+        <div className="feedTime">10시간 전</div>
+        <div className="commentInput">
+          <button className="emozi">
+            <img
+              className="smile"
+              src="/images/Jongmin/emozi.png"
+              alt="emoticon"
+            />
+          </button>
+          <input
+            className="write"
+            onChange={this.handleCommentValue}
```

Comment:
- 그렇다면 이 부분에서는 그냥 props 로 전달받은 함수를 넣어주면 될 것 같네요!
```suggestion
            onChange={this.props.handleCommentValue}
```